src/index.html

```diff
@@ -13,6 +13,49 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars__container">
+      <div class="stars stars--0">
+        <div class="stars_star"></div>
```

Comment:
`star` is defined as modifier here and further in the layout. Modifier is not an appropriate BEM entity for star.

Reply:
Fix star modifier